action/builder.go

```diff
@@ -79,23 +79,24 @@ func (b *Builder) Build() AbstractAction {
 // TODO: change envelope to *envelope
 type EnvelopeBuilder struct {
 	elp envelope
+	ab  AbstractAction
```

Comment:
If you want to keep the interface of `EnvelopeBuilder` as unchanged as possible to reduce modifications, it would be more suitable to change it to `LegacyEnvelopeBuilder`. The new `EnvelopeBuilder` may only need `SetCommon()` and `SetPayload()`.

Reply:
1. the current builder is more intuitive and easier to understand/use: `SetNonce().SetGasPrice().SetGasLimit()...`
2. SetCommon() and SetPayload() would cause too many code changes